components/Contacts.js

```diff
@@ -1,13 +1,56 @@
-import React from 'react';
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/forbid-prop-types */
 
-const Contacts = () => ({
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import uuidv4 from 'uuid/v4';
+import * as CodeableConcepts from '../util/CodeableConcepts';
+import InfoExpansion from './shared/InfoExpansion';
+import HumanName from './HumanName';
+import Organization from './Organization';
+import Period from './Period';
+import DataRow from './shared/DataRow';
+
+const defaultStyles = {
+  Contacts: {
+
+  },
+};
+
+class Contacts extends PureComponent {
   render() {
+    const { classes, contact } = this.props;
     return (
-      <>
-        {}
-      </>
+      <div className={classes.Contacts}>
+        {contact && contact.map(entry => (
+          <InfoExpansion
+            key={`InfoExpansion${uuidv4()}`}
```

Comment:
I went with `uuid` only because of another linting rule that complained about using indexes when mapping keys. I actually agree and prefer indexes, preferably with some kind of prefix involving the component name. Take a gander and feel free to override if need be.

Reply:
I'm into indexes, personally. Maybe we can turn off that linting rule.